src/kernel/pmm.zig

```diff
@@ -69,11 +69,11 @@ pub fn alloc() ?usize {
 /// Arguments:
 ///     IN addr: usize - The previously allocated address to free. Will be aligned down to the nearest multiple of BLOCK_SIZE.
 ///
-/// Error: Error || PmmBitmap.Error.
+/// Error: Error || PmmBitmap.BitmapError.
 ///     Error.NotAllocated: The address wasn't allocated.
-///     PmmBitmap.Error.OutOfBounds: The address given was out of bounds.
+///     PmmBitmap.BitmapError.OutOfBounds: The address given was out of bounds.
 ///
-pub fn free(addr: usize) (PmmBitmap.Error || Error)!void {
+pub fn free(addr: usize) (PmmBitmap.BitmapError || Error)!void {
```

Comment:
Error is ambiguous, which error set is this coming from?

Reply:
There is an error set in this file called Error, but I can rename it.